CHANGELOG

```diff
@@ -1,4 +1,18 @@
 # Changelog
+## [0.2.9] - 2020-03-25
+### Bug Fixes
+ - Fix verification errors due to outdated values: https://github.com/nmstate/nmstate/issues/853
+ - Fix missing linux bridge slaves reporting: https://github.com/nmstate/nmstate/issues/869
```

Comment:
Also have bug fixes for:

a9c4e925888b97aa802ec158d39512f4617829ea
nm sriov: Properly return when device not exists

a37ef6079c6e79dc9fce7a0ce4d45517bbe9cd8f
nm.checkpoint: fix incorrect interval when adding a timer

Reply:
I think those fix are not relevant for users. This is why I didn't include them. What do you think? @EdDev @tyll

Reply:
If they have a BZ, then it affects users, no?
The one who opened these BZ/s  (or fixed them) will know best.

Reply:
No bug reported for these two patches. I am OK no to include them.